packages/hardhat/scripts/new-dao.ts

```diff
@@ -0,0 +1,220 @@
+import hre, { ethers } from "hardhat";
+import { Contract } from "@ethersproject/contracts";
+import { Signer } from "@ethersproject/abstract-signer";
+import { GardensTemplate, Kernel } from "../typechain";
+
+const { deployments } = hre;
+
+const network = hre.network.name !== 'localhost' ? hre.network.name :  'xdai'
```

Comment:
I think the default network should be rinkeby, not xdai.

Reply:
Actually, note that by default we are using `localhost` forking the `xdai` network. To deploy to xDai you need to be precise and run: `npx hardhat deploy --network xdai`

Reply:
Oh it's confusing! Don't use a ! if you don't have too.

```
const network = hre.network.name === 'localhost' ? 'xdai' : hre.network.name
```

Reply:
Done :+1: